src/Service/AbstractJiraService.php

```diff
@@ -329,6 +329,39 @@ public function getAccountIdsByProject($projectId)
         }, []);
     }
 
+    /**
+     * Get all Jira custom fields.
+     *
+     * @return mixed
+     */
+    public function getCustomFields()
+    {
+        return $this->get('/rest/api/2/field');
+    }
+
+    /**
+     * Get custom field id by field name.
+     *
+     * @param string $fieldName
+     * @param array  $customFields
+     *
+     * @return string
+     */
+    public function getCustomFieldId($fieldName, array $customFields = null)
+    {
+        if (null === $customFields) {
+            $customFields = $this->getCustomFields();
+        }
+
+        // Get Epic name field id.
+        $customField = array_search(
+            $fieldName,
+            array_column($customFields, 'name')
+        );
+
+        return $customFields[$customField]->{'id'};
```

Comment:
Will fail successfully if a custom field with name `$fieldName` does not exist (i.e. if `false === $customField`).